kuma/users/models.py

```diff
@@ -138,9 +138,9 @@ class User(AbstractUser):
             _("Enter a valid LinkedIn URL."),
             "invalid",
         ),
-        "mozillians": validators.RegexValidator(
-            r"^https?://mozillians\.org/u/",
-            _("Enter a valid Mozillians URL."),
+        "pmo": validators.RegexValidator(
+            r"^https?://people\.mozilla\.org/u/",
```

Comment:
As above, this won't work as is, so I would suggest:
```suggestion
            r"^https?://people\.mozilla\.org/",
```